db/migrate/20180222131221_create_friendly_id_slugs.rb

```diff
@@ -0,0 +1,15 @@
+class CreateFriendlyIdSlugs < ActiveRecord::Migration[5.1]
+  def change
+    create_table :friendly_id_slugs do |t|
+      t.string   :slug,           :null => false
+      t.integer  :sluggable_id,   :null => false
+      t.string   :sluggable_type, :limit => 50
+      t.string   :scope
+      t.datetime :created_at
+    end
```

Comment:
oha, what is friendly_id doing with this table?

Reply:
Looks like it is used for keeping a history of a models' slugs https://stackoverflow.com/a/22882365/4181679

Avoiding broken urls sounds like a good idea to me, so maybe I.. should actually use `:history` and not just create a table that will not be used :D